SyntheticGeometry/SQC.lagda.md

```diff
@@ -93,10 +141,10 @@ But even more, every nonzero vector contains an invertible element.
       finite-presentation-of-A : FinitePresentation A
       finite-presentation-of-A = Instances.R/⟨xs⟩FP k xs
 
-      equiv : ⟨ A ⟩ ≃ (Spec k A → ⟨ k ⟩)
+      equiv : _ ≃ (Spec A → _)
```

Comment:
What was the intention behind reducing the type signatures around here?

Reply:
It was about avoiding this error:
```
Instance declarations with explicit arguments are never considered
by instance search, so making SyntheticGeometry.SQC._ into an
instance has no effect.
when checking that A is a valid argument to a function of type
{ℓᵢ ℓc : Level} {Instance : Type ℓᵢ}
⦃ r : Cubical.Syntax.⟨⟩.has-⟨⟩ Instance ⦄ →
Instance → Type ℓc
```

Reply:
We could also just live with the warning or figure out, why it comes up here. I don't understand, how we are making anything into an instance (claimed by the warning).

Reply:
Or I just finally fix this part of cubical, i.e. remove the instances and just replace it with projection.

Reply:
I made a PR (#903).

Reply:
Do you mean https://github.com/agda/cubical/pull/907?

Reply:
Yes. It is merged now.